meta-xt-prod-devel-rcar-control/recipes-guests/doma/doma.bbappend

```diff
@@ -45,7 +56,5 @@ do_install:append() {
         install -m 0644 ${WORKDIR}/displbe-backend.conf ${D}${sysconfdir}/systemd/system/doma.service.d
     fi
 
-    if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'false', 'true', d)}; then
-        echo "device_tree = \"/usr/lib/xen/boot/doma.dtb\"" >> ${D}${sysconfdir}/xen/doma.cfg
-    fi
+    echo "device_tree = \"/usr/lib/xen/boot/doma.dtb\"" >> ${D}${sysconfdir}/xen/doma.cfg
```

Comment:
Looks like you are doing this unconditionally. Why then you can't have this line in `doma.cfg` itself?

Reply:
Hi @lorc, I'll need to add it to 3 files:

virtio.cfg
pvr-doma-generic-h3-4x2g.cfg
pvr-doma-generic-m3-2x4g.cfg

But I accept it. Non-conditional statements should stay on a cfg-files level.

Reply:
Hi @lorc, done.